server/src/daemon/error.rs

```diff
@@ -0,0 +1,21 @@
+/// DBus Secret Service specific errors.
+/// <https://specifications.freedesktop.org/secret-service/latest/ch15.html>
+#[derive(zbus::DBusError, Debug)]
+#[zbus(prefix = "org.freedesktop.Secret.Error")]
+pub enum ServiceError {
```

Comment:
We have this error type defined in oo7 already, so one should make use of that and create a new error type that cover server side specificities